modules/installation-configuration-parameters.adoc

```diff
@@ -63,12 +63,12 @@ uses the `<metadata.name>.<baseDomain>` format.
 |`controlPlane.platform`
 |The cloud provider to host the control plane machines. This parameter value
 must match the `compute.platform` parameter value.
-|`aws`, `azure`, or `gcp`
+|`aws`, `azure`, `gcp`, or, for OpenStack, `{}`
```

Comment:
This change invalidates the descriptions for these parameters, at least for OpenStack.

Reply:
Fair point. 

@morenod Does the OSP value here definitely not match as the others do?

Reply:
when compute.platform is empty (nothing to say about it), value is {}.
when it has content, value is the name of the platform.

this is valid for aws and openstack (I cannot confirm the rest of the platforms as I havent tested them)

Reply:
@kalexand-rh So, it seems like controlPlane.platform and compute.platform can also have {} as a platform-independent value. Can you confirm this?

Reply:
> when compute.platform is empty (nothing to say about it), value is {}.
> when it has content, value is the name of the platform.

@abhinavdahiya, can you comment on the intended usage of `{}` as a `compute.platform` value in the `install-config.yaml` file? I thought it was intended to blank out the other platform variables and that the `compute.platform` and `controlPlane.platform` parameter values needed to match.

Reply:
> https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md#machine-pools

it seems like openstack does have compute level properties.

Reply:
ex: https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md#custom-machine-pools

Reply:
@abhinavdahiya So is it the case that these two fields don't necessarily have to match like the docs currently state?